### ISSUE TITLE
Calendar: convert to css-in-js

### ISSUE THREAD

**Issue by qingbolove** [OfficeDev/office-ui-fabric-react#5307]
hi,
I'd like to use calendar component, but the UI is some different with our design, how could I customize calendar UI, for example calendar day table element width and height.

I believe there should be a interface to pass in the style, like what you do to icon component, but for calendar, I could not find one, and rewrite style from outside seems could not change the layout.

```
.day {
  _width: 44px;
  height: 44px;_
  border-radius: 2px;
  display: inline-flex;
  align-items: center;
  justify-content: center;
}
```

do not work if I want to change the table element size. And I think it should be a common request for other user to customize the layout.

**Comment by dzearing** [comment]
We are tracking this conversion to add `styles` contracts to everything! We will use this issue to track the Calendar work.

**Comment by Vitalius1** [comment]
@micahgodbolt Could you kindly check with Outlook team when this conversion is planned? It is blocking our fluent project on `Calendar` which is also used in `DatePicker`.

**Comment by micahgodbolt** [comment]
The calendar conversion is being done in the new date-time package. Good news is that it's already done, bad news is that it's certainly in beta state. It's not 100% feature complete, there will be some breaking changes, and we have future plans before the 1.0 release. 

BUT hopefully this unblocks you to start styling it?

**Comment by Vitalius1** [comment]
@micahgodbolt 
I understand that the `Calendar` you are talking about is in the new package `date-time`. We could probably proceed with styling it and run the package demo to see how it works. I will get in touch with Ben and see if the current fluent toolkit is up to date with this new Calendar. Thanks for the update 👍

**Comment by micahgodbolt** [comment]
we'll be tracking the graduation of this package in #7106.

I'll close this issue down and you can follow our progress there. Note that while the conversion to  css-in-js is done, the API of the component is not yet stable.
